ev3dev/core.py

```diff
@@ -64,6 +67,10 @@
 INPUT_AUTO = ''
 OUTPUT_AUTO = ''
 
+# The number of milliseconds we wait for the state of a motor to
+# update to 'running' in the "on_for_XYZ" methods of the Motor class
+WAIT_RUNNING_TIMEOUT = 100
```

Comment:
Do we really need it? Should not it be solved by #241 and the related changes in kernel drivers (ev3dev/lego-linux-drivers#57)?

Reply:
It is, what I found is that if you do
```
    m.run_to_abs_pos(...)
    m.wait_until_not_moving()
```
The run_to_abs_pos will update the `command` file but there is some finite amount of time before the kernel updates the `state` file to `running`.  So you really have to wait until the motor starts moving before you can wait for it to stop moving.